src/libs/ReportActionsUtils.ts

```diff
@@ -320,6 +319,11 @@ function getCombinedReportActions(reportActions: ReportAction[], transactionThre
     // Filter out request and send money request actions because we don't want to show any preview actions for one transaction reports
     const filteredReportActions = [...reportActions, ...filteredTransactionThreadReportActions].filter((action) => {
         const actionType = (action as OriginalMessageIOU).originalMessage?.type ?? '';
+        const report = allReports?.[action?.reportID ?? ''];
+        const isSelfDM = report?.chatType === CONST.REPORT.CHAT_TYPE.SELF_DM;
```

Comment:
```suggestion
```
My bad! There is no `reportID` on `action`. Could not catch Copilot's shenanigans. I think we better pass the `reportID` to `getCombinedReportActions` in `OptionsListUtils` and add `reportID` as an optional parameter for `getCombinedReportActions`.